challenges/1-media-alunos/js/darktechlc/media.js

```diff
@@ -0,0 +1,7 @@
+const somaDuasNotas = (nota1, nota2) => +nota1 + nota2;
```

Comment:
Acredito que a operação `+nota1+nota2` não tenha o resultado esperado de "forçar" valores positivos. 
Você pode usar o `Math.abs()` para isso, essa função retorna o valor absoluto, sem sinais.

Reply:
Olá, ImMurderer, o intuito do `+nota1` no código é realizar a conversão explícita do argumento, no caso do "número ser passado como uma _string_", por exemplo: `'22'`, assemelhando-se ao `Number` do JS.

Reply:
Entendi, não seria melhor usar o Number, pela legibilidade?

Reply:
Certamente sim, só quis apresentar outro método/truque. ;)